consume.go

```diff
@@ -275,6 +287,16 @@ func (cmd *consumeCmd) setupClient() {
 	if cmd.verbose {
 		fmt.Fprintf(os.Stderr, "sarama client configuration %#v\n", cfg)
 	}
+	// AFAIK kafka authentication only works when the ca, cert and certkey are
+	// presented.
```

Comment:
let's pull most of this into common.go - then we don't need this differently for every command and could add a bit more ui-friendliness. e.g.:

```go
func setupTLSConfig(cert, ca, key string) (tls.Config, error) {
```
then when we setup the commands we'd use it like this:
```go
tls, err := setupTLSConfig(cmd.tlsCert, cmd.tlsCA, cmd.tlsKey)
if err != nil {
  failf("failed to setup TLS config err=%v", err)
}
if tls != nil {
  cfg.Net.TLS.Enable = true
  cfg.Net.TLS.Config = tls
}
```
so `setupTLSConfig` could return errors like `TLS cert was supplied, but CA or key is missing`and we'd keep the duplication in the command minimal. what do you think?
untested code btw :innocent: